jenkins/deploy.sh

```diff
@@ -96,6 +102,14 @@ echo "Deploy CMD: $DEPLOY_CMD"
 
 ###### Deploy the parent pom file ######
 $DEPLOY_CMD -Dfile=./pom.xml -DpomFile=./pom.xml
+PARENT_ART_ID=$(mvnEval "./" project.artifactId)
+echo "$ART_GROUP_ID:$PARENT_ART_ID:$ART_VER:pom" >> $ARTIFACT_FILE
```

Comment:
save artifact like `com.nvidia:rapids-4-spark-parent_2.12:24.02.0-SNAPSHOT:pom` into $ARTIFACT_FILE

Reply:
To elaborate on the previous comment, we don't have to register intermediate pom for an explicit check via an $ARTIFACT_FILE line.

Reply:
Updated